### ISSUE TITLE
Connect-AzureRmAccount : Fix Documentation or Fix the Cmdlet

### ISSUE THREAD

**Issue by doghousesoftware** [Azure/azure-powershell#7040]
This just does not work, even following the documentation provided.  Please fix:

Connect-AzureRmAccount -ServicePrincipal -ApplicationId $applicationId -Credential $pscredential -TenantId $tenantid
Connect-AzureRmAccount : Parameter set cannot be resolved using the specified named parameters.
At line:10 char:1
+ Connect-AzureRmAccount -ServicePrincipal -ApplicationId $applicationI ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Connect-AzureRmAccount], ParameterBindingException
    + FullyQualifiedErrorId : AmbiguousParameterSet,Microsoft.Azure.Commands.Profile.ConnectAzureRmAccountCommand

**Comment by markcowl** [comment]
Good catch!  The reference document has the correct parameter sets, but the documentation is incorrect.  Placed a PR to correct the documentation here: https://github.com/Azure/azure-docs-powershell/pull/707
